app/views/state_file/questions/federal_dependents/edit.html.erb

```diff
@@ -1,4 +1,13 @@
 <% @allow_deletion = true %>
-<% content_for :card do %>
-  <%= render "state_file/questions/federal_dependents/form" %>
+
+<% content_for :main do %>
```

Comment:
my thinking is that the edit and new pages should not have the "go back" button because it will take them to the previous question in the flow (permanent address), NOT the previous page they were on (dependents index)

Reply:
I think this is fine for now, though I imagine if we stick with this dependents flow, then dependents new & edit should link back to dependents index, and the index should link back to whatever the prior page was.